### PR TITLE
Do not serialize text attributes that are not used later

### PR DESCRIPTION
**Pull request by cubuspl42**
## Summary:

Do not serialize text attributes that are not used later

## Changelog:

[INTERNAL] [REMOVED] - Do not serialize text attributes that are not used later

### CODE REVIEW

File: packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextAttributeProps.java [hunk 1]
```diff
@@ -144,7 +137,6 @@ private TextAttributeProps() {}
   public static TextAttributeProps fromMapBuffer(MapBuffer props) {
     TextAttributeProps result = new TextAttributeProps();
 
-    // TODO T83483191: Review constants that are not being set!
```
Comment (cubuspl42) [inline packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextAttributeProps.java:147]: We're deserializing text attributes of a _fragment_. Some attributes are not handled on the fragment level but apply to the whole `Text` element. I believe there's not much more to investigate here.

Comment (cubuspl42) [reply packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextAttributeProps.java:147]: Relevant:

https://github.com/facebook/react-native/blob/27f38102973849f962801bbb184d98387f7f6022/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextAttributeProps.java#L31-L33

Comment (cubuspl42) [reply packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextAttributeProps.java:147]: The fact that we currently don't do what this comment describes is blocking me.

I'd like to...
- Split `TextAttributes` (C++) to `TextAttributes` (`<Text>`-level properties)  and `FragmentAttributes` (fragment-level properties) 
- Refactor `TextAttributeProps` (Java) to `FragmentAttributes`

Currently, one is a subset of another; not modeling them separately is at most a code smell.

I'd like them to be two slightly different sets (sharing a great common subset), so not modeling them separately is a problem.

**Comment by cubuspl42** [comment]
@mdvacca I think I will need your eyes on this one.

Please see [this thread](https://github.com/facebook/react-native/pull/42703#discussion_r1469463453).

**Comment by cubuspl42** [comment]
@mdvacca What do you think? Is this a good way forward?

**Review by mdvacca — APPROVED** [review]

File: packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextAttributeProps.java (hunk 1 above)
Comment (mdvacca) [reply packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/TextAttributeProps.java:147]: yes, this doesn't seem correct, thanks for cleaning up

**Review by mdvacca — APPROVED** [review]

**Comment by cubuspl42** [comment]
@mdvacca Let me know if I can help with making the internal build succeed

**Pull request closed without merging**